CHANGES.md

```diff
@@ -2,6 +2,9 @@
 
 This document describes the relevant changes between releases of the API model.
 
+## 0.0.309 August 08 2023
+- Add `BillingModel` resource and `BillingModelItem` type to `accounts_mgmt`
```

Comment:
Please don't update the changes file in the same MR. 

If you do want to cut a release, and bundle it in the same MR, please seperate them via the commits at least.